tests/roles/development_environment/files/pre_launch.bash

```diff
@@ -67,6 +67,8 @@ export FIP=192.168.122.20
 # check connectivity via FIP
 ping -c4 ${FIP}
 
+if [ "$CINDER_BACKEND_CONFIGURED" = "false" ]; then exit 0; fi
```

Comment:
wondering if it would be better to wrap from here to the end w/ a cinder related function, but really a minor thing wrt the flow of the script.

Reply:
I really don't know, if putting a large code block under extra indentaion would look better. I think I am inspired by Golang here, that avoids such things :)

Reply:
I would also prefer what Francesco suggested. Someone may want to add an extra thing at the end and not notice the early exit. To me the early exit is a surprising design. But it's not a blocker, if it passes CI we can merge.

Reply:
I would also prefer the cinder commands added inside a block as well since reading 20+ lines before where we want to add new code seems not usual.
This script has become the main source of validation for all resources before -> after adoption so I won't be surprised if other projects start creating resources below this.
If the indentation is an issue then we can add the following lines in a function and call it based on the CINDER_BACKEND_CONFIGURED value.

Reply:
done